packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1988,6 +2166,40 @@ function performWorkOnRoot(
       'by a bug in React. Please file an issue.',
   );
 
+  if (enableInteractionTracking) {
```

Comment:
While everything below here makes sense. This is it the right place for it.

This gets called for every scheduled work (requestIdleCallback).

We only want to call this when we’re restarting work on a root. Not when we’re resuming.

@acdlite Can probably point to the right place to do this.

Reply:
https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberScheduler.js#L1089

Reply:
Hm, I think this breaks the suspense case– (e.g. covered by test "_tracks both the temporary placeholder and the finished render for an interaction_")– since `renderRoot` is only called _once_ but `commitRoot` is potentially called multiple times.